baremetal/metal3data_manager_test.go

```diff
@@ -445,6 +447,7 @@ var _ = Describe("Metal3Data manager", func() {
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      "abc",
 					Namespace: "myns",
+					UID:       "a2e6ab49-9845-1111-1111-c74be387f57c",
```

Comment:
can't we use `m3muid` here as well?

Reply:
Correct. replaced. Also made some changes to the id